cypress/pageobject/Asset/AssetCreation.ts

```diff
@@ -31,39 +31,68 @@ export class AssetPage {
       });
   }
 
-  enterAssetDetails(
-    name: string,
-    description: string,
-    workingStatus: string,
-    qrId: string,
-    manufacturer: string,
-    warranty: string,
-    supportName: string,
-    supportPhone: string,
-    supportEmail: string,
-    vendorName: string,
-    serialNumber: string,
-    lastServicedOn: string,
-    notes: string,
-  ) {
-    cy.get("[data-testid=asset-name-input] input").type(name);
-    cy.get("[data-testid=asset-description-input] textarea").type(description);
-    cy.get("[data-testid=asset-working-status-input] li")
-      .contains(workingStatus)
-      .click();
-    cy.get("[data-testid=asset-qr-id-input] input").type(qrId);
-    cy.get("[data-testid=asset-manufacturer-input] input").type(manufacturer);
-    cy.get("[data-testid=asset-warranty-input] input").type(warranty);
-    cy.get("[data-testid=asset-support-name-input] input").type(supportName);
-    cy.get("#customer-support-phone-div").type(supportPhone);
-    cy.get("[data-testid=asset-support-email-input] input").type(supportEmail);
-    cy.get("[data-testid=asset-vendor-name-input] input").type(vendorName);
-    cy.get("[data-testid=asset-serial-number-input] input").type(serialNumber);
-    cy.clickAndTypeDate(
-      "[data-testid=asset-last-serviced-on-input]",
-      lastServicedOn,
-    );
-    cy.get("[data-testid=asset-notes-input] textarea").type(notes);
+  enterAssetDetails({
+    name,
+    description,
+    workingStatus,
+    qrId,
+    manufacturer,
+    warranty,
+    supportName,
+    supportPhone,
+    supportEmail,
+    vendorName,
+    serialNumber,
+    lastServicedOn,
+    notes,
+  }: {
+    name?: string;
+    description?: string;
+    workingStatus?: string;
+    qrId?: string;
+    manufacturer?: string;
+    warranty?: string;
+    supportName?: string;
+    supportPhone?: string;
+    supportEmail?: string;
+    vendorName?: string;
+    serialNumber?: string;
+    lastServicedOn?: string;
+    notes?: string;
+  }) {
```

Comment:
It would be better to separate out the type, so it could be reused something like `AssetCreateForm`.

This can be done later too when needed.